src/lib/platform/MSWindowsScreen.cpp

```diff
@@ -1921,9 +1912,27 @@ std::string& MSWindowsScreen::getDraggingFilename()
     return m_draggingFilename;
 }
 
-const std::string& MSWindowsScreen::getDropTarget() const
+const std::string&
+MSWindowsScreen::getDropTarget()
```

Comment:
I think such a case of cached data warrants `mutable` on `m_DropTargetPath`. This way the function can still be `const`.

Reply:
Sure. I didn't really like it that much myself, but saw it was done that way in DataDirectories class. I will make the getter const again and mark the variable as mutable then, (this is not a technique I've used a lot before either).